app/models/application_form.rb

```diff
@@ -317,13 +315,7 @@ def support_cannot_add_course_choice?
   end
 
   def maximum_number_of_course_choices
-    if FeatureFlag.active?(:apply_again_with_three_choices)
-      MAXIMUM_NUMBER_OF_COURSE_CHOICES
-    elsif apply_1?
-      MAXIMUM_PHASE_ONE_COURSE_CHOICES
```

Comment:
Delete the definition of these constants too?

Reply:
Done.